### ISSUE TITLE
Disable //src/tools/singlejar:combiners_test due to incompatible_auto_configure_host_platform

### ISSUE THREAD

**Issue by katre** [bazelbuild/bazel#9148]
Currently, `//src/tools/singlejar:combiners_test` fails to build when `--incompatible_auto_configure_host_platform` is enabled (which is #7081).

Example:
```
$ bazel test --incompatible_auto_configure_host_platform //src/tools/singlejar:combiners_test 
DEBUG: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/a85a0dacd9e20e18c47928db0f1530e0/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:103:9: rbe_ubuntu1804_java11 not using checked in configs as detect_java_home was set to True 
DEBUG: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/a85a0dacd9e20e18c47928db0f1530e0/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:103:9: rbe_ubuntu1604_java8 not using checked in configs as detect_java_home was set to True 
ERROR: While resolving toolchains for target //src/tools/singlejar:combiners_test: no matching toolchains found for types @bazel_tools//tools/cpp:toolchain_type
WARNING: errors encountered while analyzing target '//src/tools/singlejar:combiners_test': it will not be built
INFO: Analyzed target //src/tools/singlejar:combiners_test (1 packages loaded, 0 targets configured).
INFO: Found 0 test targets...
ERROR: command succeeded, but not all targets were analyzed
INFO: Elapsed time: 0.228s, Critical Path: 0.02s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
```

The fix for this is to change the target [`//:default_host_platform`](https://source.bazel.build/bazel/+/f54c2aeec89e9de176815ce68a11f3f8dbe50c93:BUILD;l=151) to inherit from `@local_config_platform//:host` instead of from `@bazel_tools//platforms:host_platform`.

However, that currently fails because in Bazel 0.28.1, `@local_config_platform//:host` was missing public visibility. This was added in ff81b0c71fc553fd3308384328908e28b6c54f3d, which is included in Bazel 0.29.0.

Therefore, I propose to disable this test in BazelCI until after Bazel 0.29.0 is released, and then re-enable it and fix the platform definition.

**Comment by katre** [comment]
Ccing @buchgr and @philwo

**Comment by katre** [comment]
0fdb1f4dc3b5639a79e5531d90c4adb1c868943d will also need to be fixed after the 0.29.0 release.

**Comment by katre** [comment]
Tests were re-enabled in 761d16c29a117751ab0312d2ca64aefc0f503c78.
